tests/fast/BlobGranuleMoveVerifyCycle.toml

```diff
@@ -3,7 +3,7 @@ testClass = "BlobGranule"
 blobGranulesEnabled = true 
 allowDefaultTenant = false
 # FIXME: re-enable rocks at some point
-storageEngineExcludeTypes = [4]
+# storageEngineExcludeTypes = [4]
```

Comment:
Just curios, Why are we commenting out the line instead of removing the line?

Reply:
I'm going to remove them.